### PR TITLE
Update client docs

### PR DESCRIPTION
**Pull request by tfalders**
Addresses SWDEV-283308.

### CODE REVIEW

**Comment by cgmb** [comment]
Not sure if this is in-scope, but it would be nice to include some example outputs (particularly for the logging).

File: docs/source/userguide_clients.rst [hunk 1]
```diff
@@ -33,13 +42,31 @@ Calling the rocSOLVER gtest client with the --help flag
 
 returns information on different flags that control the behavior of the gtests.
 
+One of the most useful flags is the --gtest_filter flag, which allows the user to choose which tests to run
```
Comment (cgmb) [inline docs/source/userguide_clients.rst:45]: Actually, I suppose the same is true here. That should be:
```
``--gtest_filter flag``
```
There's a bunch of other places where this problem occurs, too.

For consistency, I would probably just quote all command-line flags in that way, even if they're single-hyphen flags.

**Review by cgmb — APPROVED** [review]
LGTM

**Review by jzuniga-amd — APPROVED** [review]
LGTM. Just added some minor suggested changes in a PR to your branch. Have a look.

**Pull request merged**